urllib3/exceptions.py

```diff
@@ -144,6 +144,11 @@ class ResponseError(HTTPError):
     SPECIFIC_ERROR = 'too many {status_code} error responses'
 
 
+class SSLConfigurationWarning(HTTPWarning):
```

Comment:
It's not clear to me that this makes the platform less *secure*, just less functional.